python/cuml/ensemble/randomforest.pyx

```diff
@@ -352,29 +354,37 @@ class RandomForestClassifier(Base):
     Parameters
     -----------
 
-    n_estimators : number of trees in the forest. default = 10
+    n_estimators : int (default = 10)
+                   number of trees in the forest.
     handle : cuml.Handle
-        If it is None, a new one is created just for this class
-    split_algo : The type of algorithm to be used to create the trees.
-                 0 for HIST, 1 for GLOBAL_QUANTILE and 3 for SPLIT_ALGO_END.
-                 default = 0
-    bootstrap : Control bootstrapping.
+             If it is None, a new one is created just for this class.
```

Comment:
replace "class" with instance. Don't need "just":
  If it is None, a new handle is created for this instance